i18n/docs/updateinfo_en.txt

```diff
@@ -1,4 +1,7 @@
-* Adds option to set a different Wallet Service per wallet
-* Enhances  CSV generator (only desktop version)
-* Updates Cordova to version 5.3.3
-* Fixes minor bugs
+* TREZOR (for ChromeApp)
+* Improves notifications
```

Comment:
* New notification system. Reduced bandwidth usage and better handling of network disconnections.